README.md

```diff
@@ -66,20 +66,24 @@ and writes to stdout.
 #### One-off usage
 
 ```
-git show HEAD | path/to/diffr
+git show HEAD | diffr
 ```
 
 #### Integration with git
 
 Add the following section to your `.gitconfig` file:
 
 ```
-[pager]
-    log  = path/to/diffr | less -R
-    show = path/to/diffr | less -R
-    diff = path/to/diffr | less -R
+[core]
+    pager = diffr | less -R
 [interactive]
-    diffFilter = path/to/diffr
+    diffFilter = diffr
+```
+
+Alternatively, you can run from the command line:
+```
```

Comment:
Just a small nit: other code blocks have a blank line above them, so this should probably have one too.

Reply:
Other than that, the changes look good to me! 👍

Reply:
thanks for the review.